Configurations/unix-Makefile.tmpl

```diff
@@ -1435,7 +1435,8 @@ EOF
           $gen0 = platform->bin($gen0);
           return <<"EOF";
 $args{src}: $gen0 $deps \$(BLDDIR)/util/wrap.pl
-	\$(BLDDIR)/util/wrap.pl $gen0$gen_args > \$@
+	# Use $(PERL) to execute wrap.pl directly to avoid calling env
```

Comment:
I'd put it before the `return <<"EOF";` so it isn't unnecessarily written into the generated Makefiles.

Reply:
ok, need an hour - meeting

Reply:
done